packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.vue

```diff
@@ -54,6 +56,11 @@ const isChecked = computed({
     }
   }
 
+  &:has(&__input:disabled) {
+    cursor: not-allowed;
```

Comment:
Default for disabled is "cursor: default;", `not-allowed` is intended for drag-and-drop.

```suggestion
    cursor: default;
```